### ISSUE TITLE
[Accessibility], WCAG 2.1.1. The concept page tooltips are not keyboard accessible and are read two times with screen readers.

### ISSUE THREAD

**Issue by Eiskore** [NatLibFi/Skosmos#1984]
## URL address of the page where you encountered the problem
https://test.dev.finto.fi/yso/en/page/p23406 and other vocabularies concept pages

## Description of the problem
The concept pages have headings for which explanations are provided with tooltips. There are a few accessibility issues:
1.	The screen reader reads the tooltip text twice. First SR reads the heading (e.g. Tyyppi), Then the text added to the H2 title tag using aria-describedby “Valitun resurssin tyyppi" and thirdly, in the visually hidden text, “valitun resurssin tyyppi” -..

2.	Tooltips are not accessible to keyboard users. Tooltips attached to non-focusable text are not accessible to keyboard users. 

**Fix**:
1. The aria-describedby and title attribute inside the H2 tag can be removed, so that the repetition and noise flood disappear.
2. To ensure accessibility, the tooltip content must be associated with a focusable element (such as an info button or a focusable term) 

**How to test:** 1. Turn on NVDA. Navigate to some consept page heading where is a tooltip text added. When the heading is focused:
Expected result: NVDA reads the heading and the tooltip text only once

**How to test: 2**. Use keyboard only. If info buttons are added, navigate to infobutton with tab key.
Expected result: Tooltip is shown when the infobutton is focused.

Severity: moderate
Priority: medium

## Additional information (e.g. screenshots) about the problem

<img width="778" height="188" alt="Image" src="https://github.com/user-attachments/assets/0c447c5c-9e29-4180-9d10-38bee5f49ba9" />

<img width="1004" height="417" alt="Image" src="https://github.com/user-attachments/assets/1cf630d0-76f2-4918-ab3c-e5ae2a3ee4b6" />

## The browser you used when the problem appeared

Chrome, Firefox, Edge

**Comment by tinskuu** [comment]
The issue still exists.
